src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './TodoInfo.scss';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { TodosItem } from '../../types/TodosItem';
+
+type Props = {
+  doItem: TodosItem;
+};
+
+export const TodoInfo: React.FC<Props> = (props) => {
+  const { doItem } = props;
```

Comment:
Can do destructuring in arguments.
```suggestion
export const TodoInfo: React.FC<Props> = ({ doItem }) => {
```